mlengine/online-prediction/README.md

```diff
@@ -0,0 +1,9 @@
+# Cloud Machine Learning Engine - Online Prediction with Java
```

Comment:
Could you look at our other README's and add content about auth.

Reply:
Added, PTAL